api/entrypoint.sh

```diff
@@ -8,6 +8,8 @@ python manage.py collectstatic --clear --noinput || exit 1
 
 # migrate database
 python manage.py migrate || exit 1
+python manage.py makemigrations eid_server || exit 1
+python manage.py migrate eid_server || exit 1
```

Comment:
After moving the migrations to the repository, this line becomes obsolete as migrations will be applied by the previous migrate command.